### ISSUE TITLE
feature request:  Intangible crystal compatiblity with Hydraulicraft Harvester

### ISSUE THREAD

**Issue by tntblockman** [K-4U/Hydraulicraft#196]
Link to Intangible mod: http://www.curse.com/mc-mods/minecraft/241350-intangible
Feature request:
Intangible mod adds crystals,  would be nice to have the ability to automate harvesting of these crystals via Hydraulicraft Harvester <3

I'm not sure if this automation process will be added within Intangible mod later on, with it's own automation feature,  as Intangible mod is still very WIP.

**Comment by K-4U** [comment]
Waiting on version 0.1.0

**Comment by K-4U** [comment]
Closing due to inactivity of Intangible
